frontend/public/components/factory/list-page.jsx

```diff
@@ -162,7 +162,10 @@ export const FireMan_ = connect(null, {filterList: k8sActions.filterList})(
           </Link>;
         } else if (createProps.items) {
           createLink = <div className="co-m-primary-action">
-            <Dropdown buttonClassName="btn-primary" id="item-create" title={createButtonText} items={createProps.items} onChange={(name) => history.push(createProps.createLink(name))} />
```

Comment:
add generateFunction(name) to kubevirt package which will either call history.push or functions

Reply:
and localize change to a separate PR, please